christo834/CISE-assignment-1B
# refactor:package

PR:
refactor/package-json -> development branch

Added dev-dependencies
Added dependencies:
-react-icons
-react-modals
-spinners-react
-axios
}

Test:
1. Delete local node modules in frontend
2 Run npm i and npm run dev for frontend
3. Run npm i and nest start for backend
3. Test to see if pages are working as attended
